schemas/deviceShadow/ipShadow/ipShadow.json

```diff
@@ -0,0 +1,279 @@
+                "desired",
+                "reported"
+            ],
+            "title": "DeviceShadowIP"
+        },
+        "Desired": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "pairing": {
+                    "$ref": "#/definitions/Pairing"
+                },
+                "nrfcloud_mqtt_topic_prefix": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "nrfcloud_mqtt_topic_prefix",
+                "pairing"
+            ],
+            "title": "Desired"
+        },
+        "Pairing": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "state": {
+                    "type": "string"
+                },
+                "topics": {
+                    "$ref": "#/definitions/Topics"
+                }
+            },
+            "required": [
+                "state",
+                "topics"
+            ],
+            "title": "Pairing"
+        },
+        "Topics": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "d2c": {
+                    "type": "string"
+                },
+                "c2d": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "c2d",
+                "d2c"
+            ],
+            "title": "Topics"
+        },
+        "Reported": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "connection": {
+                    "$ref": "#/definitions/Connection"
+                },
+                "pairing": {
+                    "$ref": "#/definitions/Pairing"
+                },
+                "nrfcloud_mqtt_topic_prefix": {
+                    "type": "string"
+                },
+                "device": {
+                    "$ref": "#/definitions/Device"
+                },
+                "config": {
+                    "$ref": "#/definitions/Config"
+                }
+            },
+            "required": [
+                "connection",
+                "device",
+                "nrfcloud_mqtt_topic_prefix",
+                "pairing"
+            ],
+            "title": "Reported"
+        },
+        "Config": {
+            "type": "object",
+            "properties": {
+                "GPS": {
+                    "$ref": "#/definitions/Gps"
+                }
+            },
+            "title": "Config"
+        },
+        "Gps": {
+            "type": "object",
+            "properties": {
+                "enable": {
+                    "type": "boolean"
+                }
+            },
+            "required": [
+                "enable"
+            ],
+            "title": "Gps"
+        },
+        "Connection": {
```

Comment:
A lot of the definitions lack a description and examples. For example, this `connection` info is required, but it's not explained why the device must provide this information, and what `status` should contain.